metagenomics/ont/README.md

```diff
@@ -0,0 +1,60 @@
+# cheese-ont-metagenomes
+
+## Information & Purpose
+
+These are two subsampled Nanopore metagenomes sequenced from two different cheeses that were sampled at 8 and 12 weeks, respectively. The metagenomes were sequenced using the Nanopore kit [].
```

Comment:
do you need to add which nanopore kit was you used?

Reply:
ah yes I need to ask Emily, forgot I had this placeholder here

Reply:
updated